src/main/java/javaserver/DirectoryNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package javaserver;
+
+public class DirectoryNotFoundException extends Exception {
+	/**
+	 * Generated serial ID.
+	 */
+	private static final long serialVersionUID = -5356379314394533804L;
```

Comment:
The Exception class implements java's Serializable class, so Java suggests you use this ID# which tells the JVM what version of the class has been implemented. This way, if serialized classes that use a different version of the class try to deserialize, the JVM will see the different ID# and throw an appropriate error. I don't really think this scenario is likely in my situation, though. Do you think it would be cleaner to suppress the warnings and leave it out?